.github/pull_request_template.md

```diff
@@ -6,4 +6,5 @@ Fixes #
 
 - [ ] Is on the right branch
 - [ ] Documentation is up-to-date
-- [ ] Tests are up-to-date
\ No newline at end of file
+- [ ] Tests are up-to-date
+- [ ] PR title is changelog-worthy
```

Comment:
Great point, it's a good addition. I wonder if we could phrase it more elegantly though. Perhaps something like `Title accurately describes the changes`.

Reply:
The reason why I wanted to include the changelog in the sentence is that a PR can be accurately described but in a way that makes no sense in the changelog, IMO 🤔 So I feel like PR titles should be written from the perspective of the changelog, if that makes sense